### ISSUE TITLE
Model Size

### ISSUE THREAD

**Issue by sandhyacs** [ThalesAleniaSpace/DRBox_keras#7]
Hi, 
i really appreciate the work you have done. I have a concern regarding model size. I am getting model size around 300 mb which we cannot use for web and mobile app. Kindly suggest me something to reduce the model size.

**Comment by ppontisso** [comment]
Hello,
Thank you for your message.
First, you could try to reduce the number of parameters in the model, reducing number of layers and the depth of the model, especially in the VGG part.
Or you could reduce the size of the images you use as inputs, going from 512x512 to 256x256 or even lower will usually decrease the size of the model.
Finally, you can reduce the numbers of anchor boxes in the model.

**Comment by sandhyacs** [comment]
Thank you so much @ppontisso . i will try.

**Comment by sandhyacs** [comment]
one more doubt is there, in model layers, you are using lambda as one layer just after input. Why are you using that? can't we train without that?


    # Define functions for the Lambda layers below.
    ############################################################################

    def identity_layer(tensor):
        return tensor

    def input_mean_normalization(tensor):
        return tensor - np.array(subtract_mean)

    def input_stddev_normalization(tensor):
        return tensor / np.array(divide_by_stddev)

    ############################################################################
    # Build the network.
    ############################################################################

    x = Input(shape=(img_height, img_width, img_channels))

    # The following identity layer is only needed so that the subsequent lambda layers can be optional.
    x1 = Lambda(identity_layer, output_shape=(img_height, img_width, img_channels), name='identity_layer')(x)
    if not (subtract_mean is None):
        x1 = Lambda(input_mean_normalization, output_shape=(img_height, img_width, img_channels), name='input_mean_normalization')(x1)
    if not (divide_by_stddev is None):
        x1 = Lambda(input_stddev_normalization, output_shape=(img_height, img_width, img_channels), name='input_stddev_normalization')(x1)

**Comment by ppontisso** [comment]
Yes, you can train a model without the lambda layers . The identity layer is just here to make the use of the other lambda layers optional.

**Comment by sandhyacs** [comment]
Is it possible if I want train for some other object (body parts)? I tried to remove some layers but still model size is 334 mb.
